technical-reports/color/overview.md

```diff
@@ -0,0 +1,17 @@
+## Color Tokens
+
+Color tokens can be used to represent colors in different color spaces. Colors represented in tokens can then be converted to other color spaces by translation tools.
+
+Generally speaking, this module uses [[[css-color-4]]] for reference to concepts and terminology. [[[css-color-4]]] is a W3C Working Draft and is not a final specification. It is subject to change and may not be implemented in all browsers or platforms.
+
+<aside class="note" title="Why CSS Color Module Level 4?">
```

Comment:
can you change to `class="ednote"`? That's the class we use for editor's note green boxes

Reply:
will do!